src/types.ts

```diff
@@ -367,6 +367,18 @@ export class Type {
     return nullableType;
   }
 
+  /** Try to use unsigned type for according size. */
+  tryToUnsigned(): Type {
```

Comment:
Maybe
```suggestion
  toUnsignedIfApplicable(): Type {
```
or just `toUnsigned`?

Reply:
I like `toUnsigned`. Will do this changes